src/scss/content.scss

```diff
@@ -0,0 +1,102 @@
+//////////////Carousel text animation////////////////////////////
+
+.animated {
+  -webkit-animation-duration: 1.5s;
+  animation-duration: 1.5s;
+}
+
+@keyframes fadeInRight {
+
+  from {
+    opacity: 0;
+    -webkit-transform: translate3d(300px, 0, 0);
+    transform: translate3d(300px, 0, 0);
+  }
+
+  to {
+    opacity: 1;
+    -webkit-transform: none;
+    transform: none;
+  }
+}
+
+.fadeInRight {
+  animation-name: fadeInRight;
+}
+
+.bg-black-50 {
```

Comment:
Move this to separate file, like helper.scss

Reply:
Let's follow bootstrap's naming conventions: https://github.com/twbs/bootstrap/tree/v4.6.0/scss/utilities